src/commands/language.ts

```diff
@@ -0,0 +1,87 @@
+import { botCache } from "../../deps.ts";
+import { PermissionLevels } from "../types/commands.ts";
+import { createSubcommand, createCommand, getCurrentLanguage } from "../utils/helpers.ts";
+import { Embed } from "../utils/Embed.ts";
+import { db } from "../database/database.ts";
+
+const allowedLanguages = [ { id: "en_US", flag: ":flag_us:", name: "English"}, { id: "cs_CZ", flag:":flag_cz:", name: "Czech"}];
```

Comment:
This might mean you forgot to do `deno fmt **/*.ts`

Reply:
Running `deno fmt **/*.ts` changed 0 files. I am not sure I understand what is the problem here. Sorry.

Reply:
Just add breaks so the code looks nice and formatted.

Reply:
Just to clarify you want a new line like this ?
```js
const allowedLanguages = [
  { id: "en_US", flag: ":flag_us:", name: "English" },
  { id: "cs_CZ", flag: ":flag_cz:", name: "Czech" },
];
```

Reply:
Yes that's looking better.